### PR TITLE
Developing...

### PR DESCRIPTION
**Pull request by timohahaaa**
https://timohahaaa.github.io/layout_stars/report/html_report/

https://timohahaaa.github.io/layout_stars/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  background-position: center;
+}
+
+.stars--5 span:nth-child(-n+5) {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```
Comment (qoa11a) [inline src/style.css:56]: You should move it under one selector like this:
`.stars--1 span:first-child, .stars--2 span:nth-child(-n+2), ... { ...styles go here }`

**Review by qoa11a — CHANGES_REQUESTED** [review]
Almost perfect! One small fix.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 span:first-child,
+.stars--2 span:nth-child(-n+2),
+.stars--3 span:nth-child(-n+3),
+.stars--4 span:nth-child(-n+4),
+.stars--5 span:nth-child(-n+5) {
```
Comment (polosanya) [inline src/style.css:22]:
```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(-n+5) {
```
There is no need to increase the specificity

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (polosanya) [inline src/style.css:27]: Do you really need to duplicate all styles? Only background image changes, all other properties are the same

**Review by polosanya — CHANGES_REQUESTED** [review]
A few small, but important changes left 🙂

**Review by polosanya — APPROVED** [review]
Well done! 🔥

**Review by polosanya — APPROVED** [review]
Super!

**Review by qoa11a — APPROVED** [review]

**Pull request closed without merging**